tools/elf2hunk.c

```diff
@@ -276,10 +306,20 @@ static void set_error(int err)
 #define D(x)
 #define DB2(x)
 #endif
+
+#if defined(_MSC_VER)
+#define bug(fmt,...)    fprintf(stderr, fmt , __VA_ARGS__ )
+#else
 #define bug(fmt,args...)	fprintf(stderr, fmt ,##args )
+#endif
 
 static int must_swap = -1;
 
+// hack for SHN_COMMON
+static struct hunkheader *theBSS;
```

Comment:
> Original code doesn't use global variables

`static int must_swap = -1;`
is an already present global variable.

> Is there a way to avoid defining theBSS and bssIndex as globals?

Sure there always is a way. But since this file isn't included elsewhere, I wont add more time to enhace it.

Feel free to reject the pull request.